oxia/example_test.go

```diff
@@ -18,10 +18,23 @@ import (
 	"context"
 	"fmt"
 	"log"
+	"os"
+	"sync"
 	"time"
+
+	"github.com/streamnative/oxia/server"
 )
 
 func Example() {
+	dir, _ := os.MkdirTemp(os.TempDir(), "oxia-test-*")
```

Comment:
Perhaps this part can be put into a function to keep the example code shorter

Reply:
Done, PTAL